server/configuration.js

```diff
@@ -1,9 +1,11 @@
 export const setEnvironment = () => {
     const productionGraphqlEndpoint = 'https://tray.io/graphql';
     const stagingGraphqlEndpoint = 'https://staging.tray.io/graphql';
+    const productionEuGraphqlEndpoint = "https://eu1.tray.io/graphql";
     const frontendStagingGraphqlEndpoint = 'https://frontend-staging.tray.io/graphql';
 
     const appUrlProd = 'https://embedded.tray.io';
+    const appEuUrlProd = "https://embedded.eu1.tray.io";
```

Comment:
Sorry, sounds picky but the ordering of endpoint variables has become inconsistent here - `productionEuGraphqlEndpoint` is third in the section of graphql endpoints, but `appEuUrlProd` is second in the section of app endpoints. Can it be made consistent? (Again, only because this is customer facing :) )